### DISCUSSION TITLE
امکان اتصال چند سرور

### DISCUSSION THREAD

**Discussion by zZedix** [hiddify/Hiddify-Manager#207] [Q&A سوالات رایج]
سلام 
چند پیشنهاد و خواهش داشتم
ممنون از زحمات شما
آیا در امکانش هست در آینده امکان اتصال چند سرور به پنل رو هم مهیا کنید ؟
و اینکه اگر بشه برای کانکشن ها خودمون بتونیم اسم انتخاب کنیم خیلی عالی میشه
بازم تشکر

**Comment by robin98** [comment]
منظورتون اینه مثلا یه پنل هایدیفای مرکزی وجود داشته باشه و بشه چندین سرور توزیع شده و مستقل دیگه رو یکجا و از روی اون مدیریت کرد؟

**Reply by zZedix** [reply]
دقیقا

**Reply by Obeid1375** [reply]
اینجوری عالی میشه. یا حداقل بشه یه تگی بغل اسم کانفیگ ها اضافه کرد مقلا نوشت همراه اول.

**Comment by hiddify-com** [comment]
این هدفمون بوده ولی علی الحساب میتونید از relay استفاده کنید

**Reply by zZedix** [reply]
بشدت ممنون از این زحمت !
امیدوارم شما ترتیبی بدید که بتونیم اسم دلخواه و امکاه چند سروری فراهم بشه
